server/ciphersuites_1.8.go

```diff
@@ -39,6 +39,8 @@ func defaultCipherSuites() []uint16 {
 		tls.TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,
 		tls.TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,
 		tls.TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,
+		tls.TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305,
```

Comment:
After some research, and given NATS usage patterns, it does look like the preference should be for supporting for smaller clients (chipsets without the AES-NI instruction set) right now.  If someone wants to reorder preferences for GCM, they certainly can in our config. I can reorder these to place chacha first then...  sound good?